pkg/tsdb/azuremonitor/azuremonitor-datasource.go

```diff
@@ -292,15 +299,17 @@ func (e *AzureMonitorDatasource) parseResponse(queryRes *tsdb.QueryResult, data
 			default:
 				value = point.Count
 			}
-			points = append(points, tsdb.NewTimePoint(null.FloatFrom(value), float64((point.TimeStamp).Unix())*1000))
+
+			frame.SetRow(i, point.TimeStamp, value)
```

Comment:
Shouldn't timestamp be in epoch ms? Looks like old code converts from eppch seconds to ms?

Reply:
Or maybe I'm wrong? Confused by the `tsdb.NewTimePoint`...

Reply:
Oh sorry I missed this comment. Time in dataframes are now time.Time objects.

Reply:
That being said, when turned into Arrow they do become Nanosecond epochs:

https://github.com/grafana/grafana-plugin-sdk-go/blob/6561b72621d71a8efa5be36f2469e1c0eb003ada/data/arrow_column_builders.go#L393-L401